gulp/aliases.js

```diff
@@ -22,7 +22,8 @@ module.exports = (gulp) => {
     gulp.task("build", (done) => rs("clean", "compile", "webpack-compile-docs", done));
 
     // build code, run unit tests, terminate
-    gulp.task("test", ["test-dist", "karma", "isotest"]);
+    // NOTE: circle.yml runs each of these separately for better error reporting; be sure to sync
```

Comment:
"be sure to sync"? what does that mean? also, how does it produce better error reporting? does it run the next gulp task even if a previous one fails?